index.js

```diff
@@ -16,6 +17,9 @@ function resolveModules(options) {
     throwOnRootAccess = false;
   }
 
+  if (typeof options.moduleRoot === 'string') {
+    moduleRoot = options.moduleRoot;
```

Comment:
Variable declared here: https://github.com/ember-cli/amd-name-resolver/pull/12/files#diff-168726dbe96b3ce427e7fedce31bb0bcR14
Which merged in a previous PR https://github.com/ember-cli/amd-name-resolver/pull/11